src/test/unit/window-console.test.js

```diff
@@ -30,10 +30,10 @@ describe('console-accessible values on the window object', function() {
 
   it('logs a friendly message', function() {
     const log = console.log;
-    (console: Object).log = jest.fn();
+    (console: any).log = jest.fn();
```

Comment:
Probably, but I'd prefer not to take it on in this patch, since it's out of scope and involves bigger changes. The type-safety is unchanged with this change.